import logging
logger = logging.getLogger('example_logger')
logger.warning('This is a warning')except Exception as e:
logging.error("Exception occurred", exc_info=True)import logging
a = 5
b = 0
try:
c = a / b
import logging
name = 'John'
error(f'{name} raised an error')
